README.md

```diff
@@ -1,3 +1,23 @@
+# Table of Content
+
+- [CodeLabz](#codelabz)
```

Comment:
Try adding like this instead

1. [CodeLabz](#codelabz)
2. [Deployed on](#deployed-on)
  1. [User Guide](#user-guide)
      1. [How to Setup](#how-to-setup)
      2. [How to Use](#how-to-use)
  2. [Set-up-Firebase](#set-up-firebase)
      1. [How to Deploy Firebase Cloud Functions](#how-to-deploy-firebase-cloud-functions)